www/README.md

```diff
@@ -1,16 +1,55 @@
-# React documentation site
-
-To get setup:
-```
-yarn install
-```
-
-To run the dev server:
-```
-yarn run dev
-```
-
-To clear the local cache:
-```
-yarn run reset
-```
\ No newline at end of file
+# React — documentation site
+
+## Getting started
+
+### Prerequisites
+
+1. Git
+1. Node: install version 7.6 or greater
+1. Yarn: `npm i -g yarn` to install it globally via NPM
+1. A clone of the [React repo](https://github.com/facebook/react) on your local machine
+1. A fork of the repo (for any contributions)
+
+### Installation
+
+1. `cd react` to go into the project root
+1. `yarn` to install the main NPM dependencies
+1. `cd www` to go into the website project
+1. `yarn` to install the website's NPM dependencies
+
+### Running locally
+
+1. `yarn dev` to start the hot-reloading development server powered by Gatsby
+1. `open http://localhost:8000` to open the site in your favourite browser
+
+## Contributing
+
+### Create a branch
+
+1. `git checkout master` from any folder in your local react repository
+1. `git pull origin master` to ensure you have the latest main code
+1. `git checkout -b wwwTheNameOfMyChange` (replacing `TheNameOfMyChange` with a suitable name) to create a branch
+
+### Make the change
+
+1. Follow the "Running locally" instructions
+1. Save the files and check in the browser
+  1. Any React components in `src/components` or `src/templates` will hot-reload
```

Comment:
Had one more suggestion :-)

"Edits to markdown files in `/docs` and to React components in…"